charts/spire/charts/spire-server/templates/controller-manager-webhook.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if eq (.Values.controllerManager.enabled | toString) "true" }}
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: {{ include "spire-controller-manager.fullname" . }}-webhook
+webhooks:
+  - admissionReviewVersions: ["v1"]
+    clientConfig:
+      service:
+        name: {{ include "spire-controller-manager.fullname" . }}-webhook
+        namespace: {{ .Release.Namespace }}
+        path: /validate-spire-spiffe-io-v1alpha1-clusterfederatedtrustdomain
+    failurePolicy: Fail
+    name: vclusterfederatedtrustdomain.kb.io
+    rules:
+      - apiGroups: ["spire.spiffe.io"]
+        apiVersions: ["v1alpha1"]
+        operations: ["CREATE", "UPDATE"]
+        resources: ["clusterfederatedtrustdomains"]
+    sideEffects: None
+  - admissionReviewVersions: ["v1"]
+    clientConfig:
+      service:
+        name: {{ include "spire-controller-manager.fullname" . }}-webhook
+        namespace: {{ .Release.Namespace }}
+        path: /validate-spire-spiffe-io-v1alpha1-clusterspiffeid
+    failurePolicy: Fail
+    name: vclusterspiffeid.kb.io
```

Comment:
Weird name, where is this coming from?

Reply:
Spire examples. Probably some boilerplating when they started the spire-controller-manager. Will check with them regarding this kb.io. I has similar thoughts.